pip/commands/list.py

```diff
@@ -237,8 +244,9 @@ def output_package_listing(self, packages, options):
         elif options.list_format == 'freeze':
             for dist in packages:
                 if options.verbose >= 1:
-                    logger.info("%s==%s (%s)", dist.project_name,
-                                dist.version, dist.location)
+                    logger.info("%s==%s (%s) (%s)", dist.project_name,
```

Comment:
I'm not sure it makes sense to include the installer (and the location, fwiw) in the `freeze` format.